src/main/java/org/seqdoop/hadoop_bam/FileVirtualSplit.java

```diff
@@ -79,15 +88,32 @@ public FileVirtualSplit(Path f, long vs, long ve, String[] locs) {
 	public void setStartVirtualOffset(long vo) { vStart = vo; }
 	public void   setEndVirtualOffset(long vo) { vEnd   = vo; }
 
+	public long[] getIntervalFilePointers() {
```

Comment:
It would be helpful to have some comments in this file saying what these are (i.e., BAMFileSpan chunk start/stop pairs for the split), and that this will return null if no query interval was specified, etc.